README.md

```diff
@@ -1 +1,5 @@
-# code-review-project
```

Comment:
Pull Request should have a summary describing introduced changes in more detail, kindly add a description to your PR

Reply:
Great job @lucascomamala 🤝

- [ ] Well explained and polite comment. However, it would also be human-friendly if you used emojis. ✔️